docs/docsite/rst/roadmap/COLLECTIONS_6.rst

```diff
@@ -57,3 +57,10 @@ Ansible 6.x minor releases will occur approximately every three weeks if changes
 
 
 For more information, reach out on a mailing list or a chat channel - see :ref:`communication` for more details.
+
+Planned work
```

Comment:
Do we have GH Issues (or PRs) for this work, might be good to include links.

Reply:
I've added a generic link to the community-topics issue (377e43d5e0cda574e2081839ef30dd390771cdd3). If we want to make this a precise list, we can probably only merge this shortly before Ansible 6 is ready. I'm even considering throwing out this session so we can first fix the timeframe before concentrating on details of what to do.

Reply:
Ok, this is good enough for the moment. Thanks for adding the extra details